core-java-modules/core-java-11-2/src/test/java/com/baeldung/httpclient/ssl/HttpClientSSLBypassUnitTest.java

```diff
@@ -4,29 +4,84 @@
 import org.junit.jupiter.api.Test;
 
 import java.io.IOException;
+import java.net.Socket;
 import java.net.URI;
+import java.net.URISyntaxException;
 import java.net.http.HttpClient;
 import java.net.http.HttpRequest;
 import java.net.http.HttpResponse;
+import java.security.KeyManagementException;
+import java.security.NoSuchAlgorithmException;
+import java.security.SecureRandom;
+import java.security.cert.X509Certificate;
 import java.util.Properties;
 
+import javax.net.ssl.SSLContext;
+import javax.net.ssl.SSLEngine;
+import javax.net.ssl.TrustManager;
+import javax.net.ssl.X509ExtendedTrustManager;
+
 public class HttpClientSSLBypassUnitTest {
 
     @Test
-    public void whenHttpsRequest_thenCorrect() throws IOException, InterruptedException {
+    public void whenHttpsRequest_withDisabledHostName_thenCorrect() throws IOException, InterruptedException {
```

Comment:
can we rename this to, givenDisableUsingJVMProperty_whenByPassCertificationVerification_thenCorrect()